packages/firebase/index.d.ts

```diff
@@ -7682,11 +7682,23 @@ declare namespace firebase.firestore {
      * causes unexpected behavior when using a timestamp from a snapshot as a
      * part of a subsequent query.
      *
-     * So now Firestore returns `Timestamp` values instead of `Date`, avoiding
-     * this kind of problem.
-     *
-     * To opt into the old behavior of returning `Date` objects, you can
-     * temporarily set `timestampsInSnapshots` to false.
+     * Now, Firestore returns `Timestamp` values for all timestamp values stored 
+     * in Cloud Firestore instead of system `Date` objects, avoiding this kind 
+     * of problem. Consequently, you must update your code to handle `Timestamp`
+     * objects instead of `Date` objects.
+     *
+     * If you wish to TEMPORARILY opt into the old behavior of returning  `Date`
```

Comment:
WDYT about making these words bold instead of capitalizing them?  This feels a little shouty.

Also, suggest "want" instead of wish.